packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -751,23 +751,13 @@ export const Table = <
     ],
   );
 
-  const headerRef = React.useRef<HTMLDivElement>(null);
-  const bodyRef = React.useRef<HTMLDivElement>(null);
+  const tableRef = React.useRef<HTMLDivElement>(null);
 
   const { scrollToIndex, tableRowRef } = useScrollToRow<T>({ ...props, page });
   const columnRefs = React.useRef<Record<string, HTMLDivElement>>({});
   const previousTableWidth = React.useRef(0);
   const onTableResize = React.useCallback(
     ({ width }: DOMRectReadOnly) => {
-      // Handle header properties, regardless of whether the table is resizable
-      setHeaderScrollWidth(headerRef.current?.scrollWidth ?? 0);
-      setHeaderClientWidth(headerRef.current?.clientWidth ?? 0);
-
-      // Handle table properties, but only when table is resizable
-      if (!isResizable) {
-        return;
-      }
```

Comment:
I think we should move this back here. It feels more clear than having it inside the ref

Reply:
Sounds good, brought back (https://github.com/iTwin/iTwinUI/pull/1891/commits/24c3d1b15d1b4ca0dccf110eb8e022c5108ba60a).

Reply:
You can remove it from the ref now. 
![image](https://github.com/iTwin/iTwinUI/assets/9084735/f43dede9-9d96-4e10-b476-439d496ded4b)

Reply:
I thought to keep it in both places to be safer. But yes, one is sufficient.

Removed that if statement (https://github.com/iTwin/iTwinUI/pull/1891/commits/730397d6ee1fac7d0275263ad1f15c9e3c6205fa).